source/codegen/metadata/nitclk/functions.py

```diff
@@ -1,606 +1,15 @@
-            'description': '\nThis function queries the value of an NI-TClk ViString attribute. You\nmust provide a ViChar array to serve as a buffer for the value. You pass\nthe number of bytes in the buffer as bufSize. If the current value of\nthe attribute, including the terminating NULL byte, is larger than the\nsize you indicate in bufSize, the function copies bufSize minus 1 bytes\ninto the buffer, places an ASCII NULL byte at the end of the buffer, and\nreturns the array size that you must pass to get the entire value. For\nexample, if the value is "123456" and bufSize is 4, the function places\n"123" into the buffer and returns 7. If you want to call\nniTClk_GetAttributeViString just to get the required array size, pass 0\nfor bufSize and VI_NULL for the value.\n'
-        },
-        'parameters': [
-            {
-                'cppName': 'session',
-                'direction': 'in',
-                'documentation': {
-                    'description': 'session references the sessions being synchronized.'
-                },
-                'grpc_type': 'nidevice_grpc.Session',
-                'name': 'session',
-                'type': 'ViSession'
-            },
-            {
-                'cppName': 'channelName',
-                'direction': 'in',
-                'documentation': {
-                    'description': 'Pass VI_NULL or an empty string'
-                },
-                'grpc_type': 'string',
-                'name': 'channelName',
-                'type': 'ViConstString'
-            },
-            {
-                'cppName': 'attributeId',
-                'direction': 'in',
-                'documentation': {
-                    'description': '\nThe ID of the attribute that you want to get Supported Attributes\nNITCLK_ATTR_SYNC_PULSE_SOURCE\nNITCLK_ATTR_SYNC_PULSE_CLOCK_SOURCE\nNITCLK_ATTR_EXPORTED_SYNC_PULSE_OUTPUT_TERMINAL\n'
-                },
-                'grpc_type': 'NiTClkAttribute',
-                'name': 'attributeId',
-                'type': 'ViAttr'
-            },
-            {
-                'cppName': 'bufSize',
-                'direction': 'in',
-                'documentation': {
-                    'description': '\nThe number of bytes in the ViChar array that you specify for the value\nparameter\n'
-                },
-                'grpc_type': 'sint32',
-                'include_in_proto': False,
-                'is_size_param': True,
-                'name': 'bufSize',
-                'type': 'ViInt32'
-            },
-            {
-                'cppName': 'value',
-                'direction': 'out',
-                'documentation': {
-                    'description': 'The value that you are getting'
-                },
-                'grpc_type': 'string',
-                'name': 'value',
-                'size': {
-                    'mechanism': 'ivi-dance',
-                    'tags': [
-                        'strlen-bug'
```

Comment:
For 'GetAttributeVIString', we'll need this 'strlen-bug' tag back. To add it, in hapigen's grpc_device/functions/functions.hapigen update 'get attribute vi string' to have the following:

```
'parameters': {
  'value': {
     'array': {
                    'tags': ['strlen-bug'],
     },
  }